src/Composer/PatchRemoveCommand.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace szeidler\ComposerPatchesCLI\Composer;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Composer\Json\JsonFile;
+use Composer\Json\JsonManipulator;
+
+class PatchRemoveCommand extends PatchBaseCommand {
+
+  protected function configure() {
+    $this->setName('patch-remove')
+      ->setDescription('Remove a from a composer patch file.')
+      ->setDefinition([
+        new InputArgument('package', InputArgument::REQUIRED),
+        new InputArgument('description', InputArgument::REQUIRED)
+      ]);
+
+    parent::configure();
+  }
+
+  protected function interact(InputInterface $input, OutputInterface $output) {
+    $dialog = $this->getHelperSet()->get('dialog');
```

Comment:
Maybe this dialog (and following) should be put as a `Trait` so can be shared between the two `patch-add` and `patch-remove` classes?
Or else have a common ancestor like `PatchFileManipulatorAbstractCommand` ?

Having a common ancestor could also help for adding/removing the patch with sharing all the options, because basically the two commands are very very similar.

Reply:
Thanks for the suggestions. Let's do it on a separate issue, after this feature has been committed. Because it has also an effect on the command that is not part of this issue/pull request.